### PR TITLE
adding pages and routing

### PR DESCRIPTION
**Pull request by misalazeem**
- Added Pages
- Added Routing
- Made website responsive for mobile
- Added mobile hamburger

### CODE REVIEW

File: src/components/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          Voluptates at.
+          Minima sint labore sed debitis sunt error iste dolor ullam blanditiis, minus nihil.
+          Voluptates at.
+          Minima sint labore sed debitis sunt error iste dolor ullam blanditiis, minus nihil.
+          Voluptates at., tempora tempore ex.
+          Non iste obcaecati doloremque hic tenetur iusto, modi ab sunt soluta velit. , illum velit?
+        </p>
+      </div>
+    </>
+  );
```
Comment (Munsa1) [inline src/components/Home.jsx:23]: - [ ] Great job creating the home page, however, our design has to follow the given wireframe. Kindly adjust the `Welcome to our page `  heading to match the given layout

Your design
image

Given wireframe
image

Kindly align the heading to the left as is in the given design

File: src/components/calculator.jsx [hunk 2]
```diff
@@ -18,11 +18,14 @@ function Calculator() {
...
-          <NumericKeys clickEvent={clickEvent} />
+      <div className="section">
+        <h2>Lets Do some Math</h2>
+        <div className="calculator">
+          <OutputScreen state={current} />
+          <div className="keys">
+            <OperationKeys clickEvent={clickEvent} />
+            <NumericKeys clickEvent={clickEvent} />
+          </div>
         </div>
```
Comment (Munsa1) [inline src/components/calculator.jsx]: - [ ] Again here kindly align the calculator UI so as to match the given wireframe. Notice how the calculator UI aligns with the `links` at the top to the right.

Your design
image

Given wireframe
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @misalazeem, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Nice UI Design :heavy_check_mark: 
Used React Router Dom For Routing Pages :heavy_check_mark: 
Descriptive PR :heavy_check_mark: 
Responsive Web Design :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @misalazeem 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Designed app according to the given wireframe ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**